libs/entitlement/src/main/java/org/elasticsearch/entitlement/bootstrap/EntitlementBootstrap.java

```diff
@@ -154,7 +154,7 @@ private static void exportInitializationToAgent() {
         EntitlementInitialization.class.getModule().addExports(initPkg, unnamedModule);
     }
 
-    private static String findAgentJar() {
+    public static String findAgentJar() {
```

Comment:
I'm not sure why this became public, but I presume that will become clear in an upcoming PR.

Reply:
This can be package private, right? The TestEntitlementBootstrap can be in the same entitlement bootstrap package (split packages are ok when using the classpath)

Reply:
Yes sorry this change slipped in (see the linked PR).
If split packages are OK for tests (as we use the classpath) I will change them the package of `TestEntitlementBootstrap` etc. and make this (and other method/classes) package-private then.